App/features/photos/sagas.ts

```diff
@@ -22,16 +23,49 @@ function * toggleStorage(action: ActionType<typeof PreferencesActions.toggleStor
   }
 }
 
-function * nodeOnline() {
-  while (yield take([getType(TextileEventsActions.nodeOnline), getType(TextileEventsActions.stopNodeAfterDelayCancelled)])) {
+function * bootstrapPhotoProcessing() {
+  const QUEUE_CONCURRENT = 1
+  const { watcher, addTaskChannel } = yield createQueue(photoHandler, QUEUE_CONCURRENT)
+  yield fork(watcher)
+  yield call(processPending, addTaskChannel)
```

Comment:
Before we allow any camera roll querying to happen, we want to add any still-pending photos to the queue first.